### ISSUE TITLE
Update Maven dependencies as far as possible

### ISSUE THREAD

**Issue by occupant23** [Xceptance/neodymium-library#138]

**Comment by occupant23** [comment]
I just tried your support for Cucumber 6 JVM and mvn. The implementations seems to work, but we have to use another way to configurate it. The old way via system property variables as stated in the allure docu (https://docs.qameta.io/allure/#_cucumber_jvm) is not supported for Cucumber 6 anymore (https://github.com/cucumber/cucumber-jvm/pull/1958).

We have to update the child projects and our documentation.

 I found the following two possibilities:
Option one wihin the testcase e.g.:
```Java
@CucumberOptions(
features = "src/test/java/template/cucumber/features", glue = “template",
plugin ={“io.qameta.allure.cucumber6jvm.AllureCucumber6Jvm”})
public class RunAllFeaturesTest
{
}
```
Option two within the src/test/resources/cucumber.properties file:
```
cucumber.plugin=io.qameta.allure.cucumber6jvm.AllureCucumber6Jvm
```
